addons/project/views/project_views.xml

```diff
@@ -1465,7 +1471,7 @@
                                     </div>
                                     <div class="oe_kanban_bottom_right" t-if="!selection_mode">
                                         <field name="kanban_state" widget="state_selection" groups="base.group_user"/>
-                                        <t t-if="record.user_ids.raw_value"><field name="user_ids" widget="many2many_avatar_user"/></t>
+                                        <t t-if="record.user_ids.raw_value"><field name="user_ids" t-attf-class="{{ record.user_ids.raw_value.length > 0 ? 'ps-2' : '' }}" widget="many2many_avatar_user"/></t>
```

Comment:
I guess this is no more needed as we decided to go with the 6px 🙂

Reply:
Indeed 🙄 
Sorry...